ark-cli/USAGE.md

```diff
@@ -74,14 +83,18 @@ $ ark-cli list ~/Pictures/
 ```
 
 But it's a bit boring and doesn't really tell anything, right? Various flags should be used to gain more knowledge about your collections of resources:
-* `--entry=id|path|both|link` to show the path,the id or both of a resource
-* `--modified` to show or not the last modified timestamp of a resource
-* `--tags=true` to show or not the tags for every resource
-* `--scores=true` to show or not the scores for every resource
-* `--sort=asc|desc` to sort resources by asc or dsc order of scores
-* `--filter=query` to filter resources by their tags
+
+- `--entry=id|path|ip|link` to show the path,the id or both of a resource
+- `--modified` to show or not the last modified timestamp of a resource
+- `--tags=true` to show or not the tags for every resource
+- `--scores=true` to show or not the scores for every resource
```

Comment:
Would be great to make these two options simple flags, too:
* `--tags` should mean `--tags=true`, and `false` default
* `--scores` should mean `--scores=true`, and `false` default

Reply:
Updated documentation for these 2 flags